packages/peregrine/lib/talons/CartPage/ProductListing/EditModal/useEditModal.js

```diff
@@ -0,0 +1,16 @@
+import { useCallback } from 'react';
+import { useAppContext } from '@magento/peregrine/lib/context/app';
```

Comment:
Copy/pasta'd this from another talon, but I am concerned this might be needed for something (extensibility maybe?). There are 80 other occurrences of `@magento/peregrine` in the Peregrine package. Can we keep as-is and create a JIRA ticket to investigate/fix?

Reply:
I actually made this change in 1522b8d, but worth creating a ticket to address the rest of the app.